velox/common/caching/SsdFile.cpp

```diff
@@ -119,8 +119,16 @@ SsdFile::SsdFile(const Config& config)
   filesystems::FileOptions fileOptions;
   fileOptions.shouldThrowOnFileAlreadyExists = false;
   fileOptions.bufferIo = !FLAGS_velox_ssd_odirect;
-  writeFile_ = fs_->openFileForWrite(fileName_, fileOptions);
-  readFile_ = fs_->openFileForRead(fileName_, fileOptions);
+
+  try {
+    writeFile_ = fs_->openFileForWrite(fileName_, fileOptions);
+    readFile_ = fs_->openFileForRead(fileName_, fileOptions);
+  } catch (std::exception& e) {
+    ++stats_.openFileErrors;
+    VELOX_SSD_CACHE_LOG(ERROR)
+	        << fmt::format("Error opening ssd file {}", e.what());
```

Comment:
our linter is still complaining :(
```suggestion
        << fmt::format("Error opening ssd file {}", e.what());
```

Reply:
is there a tab \t in the middle of blank spaces?